library/std/src/sys/wasi/thread.rs

```diff
@@ -178,6 +179,17 @@ impl Thread {
     }
 }
 
+cfg_if::cfg_if! {
```

Comment:
Is there a reason not to put this alongside the definition of Thread, i.e., in that same cfg_if?

Reply:
Moved it to the same cfg_if